sirepo/template/srw.py

```diff
@@ -379,6 +412,8 @@ def extract_report_data(sim_in):
         return _extract_trajectory_report(dm.trajectoryReport, out.filename)
     if r == _SIM_DATA.EXPORT_RSOPT:
         return out
+    if out.get("check_backup"):
+        out.filename = _check_backup(out)
```

Comment:
Use `_best_data_file(out.filename)`. This is not really about check_backup, but about picking between a primary and second file.